src/main/java/com/synopsys/integration/alert/provider/blackduck/descriptor/BlackDuckDistributionUIConfig.java

```diff
@@ -75,12 +75,11 @@ public BlackDuckDistributionUIConfig(final BlackDuckProvider provider) {
 
     private Collection<String> validateConfiguredProject(final FieldValueModel fieldToValidate, final FieldModel fieldModel) {
         final Collection<String> configuredProjects = Optional.ofNullable(fieldToValidate.getValues()).orElse(List.of());
-        final String filterByProject = fieldModel.getField(BlackDuckDescriptor.KEY_FILTER_BY_PROJECT).flatMap(field -> field.getValue()).orElse(null);
-        final String projectNamePattern = fieldModel.getField(BlackDuckDescriptor.KEY_PROJECT_NAME_PATTERN).flatMap(field -> field.getValue()).orElse(null);
-        final boolean trueTextPresent = Boolean.TRUE.toString().equalsIgnoreCase(filterByProject);
-        final boolean falseTextPresent = Boolean.FALSE.toString().equalsIgnoreCase(filterByProject);
+        final String filterByProject = fieldModel.getField(BlackDuckDescriptor.KEY_FILTER_BY_PROJECT).flatMap(FieldValueModel::getValue).orElse(null);
```

Comment:
Rather than doing the `Boolean.parseBoolean` after the fact, you could add 
```java
.map(Boolean::parseBoolean).orElse(false);
```
after the flatMap